src/App/Fossa/Monorepo.hs

```diff
@@ -44,9 +44,8 @@ monorepoScan MonorepoAnalyzeConfig{..} = withWigginsBinary $ \binaryPaths -> do
           monorepoMetadata
 
   logInfo "Running monorepo scan"
-  stdout <- context "Monorepo" $ runExecIO $ runWiggins binaryPaths wigginsOpts
```

Comment:
Interesting.  It looks like after this change this anonymous function no longer depends on `Lift IO`, but the whole function still does because `withWigginsBinary` still goes through `IO`.  In the long run it would be cool if we pushed the exec commands up the stack to both load and run the binary.

No action here.  Just glad to see extra `runExecIO` commands getting pulled out of the codebase. :D